vendor/modules.txt

```diff
@@ -611,8 +610,8 @@ golang.org/x/text/unicode/norm
 # golang.org/x/time v0.0.0-20220224211638-0e9765cccd65
 ## explicit
 golang.org/x/time/rate
-# golang.org/x/tools v0.1.10-0.20220218145154-897bd77cd717
-## explicit; go 1.17
+# golang.org/x/tools v0.6.0
+## explicit; go 1.18
```

Comment:
Should this reference to 1.18 be there ?

Reply:
Yes.